src/utils.ts

```diff
@@ -131,9 +131,9 @@ export async function submitResponseToSupabase(
   }
 }
 
-export const removeStartAndEndQuotes = (str: string | null) => {
+export function removeStartAndEndQuotes(str: string | null) {
```

Comment:
@ellipsis-dev add a docstring. Make sure to run the build command to make sure everything works.